schema/domain/barnevern/delmal.v1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$id": "https://api.fiks.ks.no/innsyn/schema/domain/barnevern/delmal.v1.json",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "title": "Barnevern delmal V1",
+  "description": "Delmål for tiltaksplan",
+  "type": "object",
+  "required": [
+    "id",
+    "beskrivelse"
+  ],
+  "properties": {
+    "id": {
+      "description": "Identifikator for delmål",
```

Comment:
Denne IDen har slik det er nå ikke noen betydning for logikken i vår kode, men er ment for at mottaker skal enklere kunne holde orden. Burde dette kommet frem tydeligere?